test/signedTx/vsizeEstimationErrors.ts

```diff
@@ -36,14 +40,16 @@ describe(`Dimension estimation errors`, function() {
   // set to `true` if we want the test to fail if the error is *smaller* than expected
   const strictErrorBounds = false;
 
-  const getExpectedInputErrors = (inputType: any, inputCount: any, outputType: any) => {
+  const getExpectedInputErrors = (inputType: string, inputCount: number): [low: number, high: number] => {
     switch (inputType) {
       case UnspentTypeScript2of3.p2sh:
         return [0, 5 * inputCount];
       case UnspentTypeScript2of3.p2shP2wsh:
       case UnspentTypeScript2of3.p2wsh:
       case UnspentTypeScript2of3.p2tr:
         return [0, inputCount];
+      case UnspentTypeP2shP2pk:
+        return [0, 3 * inputCount];
```

Comment:
this basically means that we overestimate the actual size to up to three vBytes per input